File: vue/src/constants/reserved.ts
/**
 * constants/reserved.ts - 前端保留路径常量
 * 职责：与后端 pkg/constants/reserved.go 保持同步，防止保留路径被误识别为 tenantId
 * 对外暴露：RESERVED_TENANT_IDS
 */

// 注：favicon.ico / robots.txt 含特殊字符，无法通过 tenantId 正则，无需列入
export const RESERVED_TENANT_IDS = new Set([
  'admin',
  'api',
  'assets',
  'static',
  'health',
  'sitemap',
])

File: backend/pkg/constants/reserved.go
// Package constants 系统常量定义
//
// 职责：定义租户 ID 保留词列表
// 对外接口：ReservedTenantIDs
package constants

// ReservedTenantIDs 系统保留的路径段，禁止用作 tenant_id
var ReservedTenantIDs = map[string]struct{}{
	"admin":       {},
	"api":         {},
	"assets":      {},
	"static":      {},
	"health":      {},
	"favicon.ico": {},
	"robots.txt":  {},
	"sitemap":     {},
}
